events/README.md

```diff
@@ -247,6 +248,24 @@ Type(s) of lane used or blocked `vehicle_blocked_lane_types` by the vehicle perf
 
 [Top][toc]
 
+### Payment Type
+
+Strings used to indicate how a curb user paid for a curb event.
+
+| Name              | Description                                            |
+| ----------------- | ------------------------------------------------------ |
+| `cash`            | Bills or coins at a meter. |
+| `credit_card`     | Visa, Mastercard, etc at a meter. |
+| `membership_card` | A card used at a meter to pay via a corporate membership or loyalta program, etc. |
+| `mobile`          | Paid via a mobile app. |
+| `sms`             | Paid via text message. |
+| `billing`         | Curb user will be billed for usage at a later time. |
+| `permit`          | Curb user has a permit allowing them to use the curb without payment. |
+| `courtesy`        | At a curb that normally requires payment this event for some reason did not. |
```

Comment:
`voucher` and `test` might also be useful options from MDS.
https://github.com/openmobilityfoundation/mobility-data-specification/blob/main/modes/passenger-services.md#fare-attributes